script.js

```diff
@@ -71,7 +83,7 @@ let mapConfig = {
   states: true,
   cities: true,
   counties: true,
-  rentStrikes: true
```

Comment:
Just a small thing, but it's best not to include formatting changes, since it makes it harder to review.

Reply:
good point @river-honer I got a little ahead of myself. The frontend build PR includes code formatting using `prettier` as a pre-commit hook, so there won't be a need for code formatting commits in the future.